swimrankings.py

```
@@ -1,136 +1,215 @@
+                    raise RuntimeError("Header div not found!")
+
+                img = header_div.find('img', attrs={'align': 'top'})
+                if not img:
+                    raise RuntimeError("Img in header div not found!")
+
+                if img['src'] == 'images/gender1.png':
+                    return Gender(0)
+                else:
+                    return Gender(1)
+
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)